src/lib/i18n/settings.ts

```diff
@@ -0,0 +1,13 @@
+export const fallbackLng = "en"
+export const languages = [
+  fallbackLng,
+  "ar"
+]
+export const localePrefix = "always"
+export const LOCALE_COOKIE = "NEXT_LOCALE"
+
+export const intlConfig = {
+  locales: languages,
+  defaultLocale: fallbackLng,
+  localeDetection: true,
+}
```

Comment:
[`defineRouting`](https://next-intl.dev/docs/routing#define-routing) helps to define all relevant config in a type-safe way.